README.md

```diff
@@ -29,6 +29,7 @@ Currently, the minimum supported Rust version is `1.48.0`.
 ### Development Branch
 
 - [Book](https://gtk-rs.org/gtk4-rs/git/book)
+- [Examples](https://github.com/gtk-rs/gtk4-rs/tree/master/examples)
```

Comment:
Maybe we should drop the whole development/stable difference here and have something like

```
## Documentation
- Book : [Stable]/[Development]
- Docs: [Stable]/[Development]
- [Examples]()

```
The docs link would be gtk-rs.org/gtk4-rs/docs/git & stable once we have a stable release.

Reply:
Good idea!